mesonbuild/scripts/gettext.py

```diff
@@ -66,6 +66,16 @@ def gen_gmo(src_sub, bld_sub, langs):
                                '-o', os.path.join(bld_sub, l + '.gmo')])
     return 0
 
+def generate_mo(src_sub, bld_sub, pkgname, langs):
+    for l in langs:
+        srcfile = os.path.join(src_sub, l + '.po')
+        outfile = os.path.join(bld_sub, l, 'LC_MESSAGES',
+                               pkgname + '.mo')
+        os.makedirs(os.path.dirname(outfile), exist_ok=True)
```

Comment:
Is this safe to assume? Can we create subdirs in the builddir this way without worrying? I guess in the real world this _should_ never happen in this case, but just to be sure^^